src/main.rs

```diff
@@ -949,6 +957,24 @@ async fn main() {
         );
     }
 
+    let mut runner = RunnerBuilder::new(opts.server, opts.token, dir.path(), jobs)
+        .version(env!("CARGO_PKG_VERSION"))
+        .revision(
+            option_env!("VERGEN_GIT_SHA")
+                .map(|sha| {
+                    if option_env!("VERGEN_GIT_DIRTY") == Some("true") {
+                        format!("{}-dirty", sha)
+                    } else {
+                        sha.to_string()
+                    }
+                })
+                .unwrap_or_else(|| "-".to_string()),
+        )
+        .architecture("gitlab-runner-rs")
```

Comment:
I guess reporting the current architecture is not much valuable here and this is a copy and paste from `demo-runners.rs`, but `gitlab-runners-rs` seems wrong anyway.

Is it mandatory to set it?